launches/launch.vehicle.py

```diff
@@ -51,12 +51,13 @@ def generate_launch_description():
 
                     # PERCEPTION
                     *perception_launch_entities,
+                    traffic_light_detector,
```

Comment:
could you move this to the perception launch file so it would be included in "perception_launch_entities" already? I think it would make things a bit cleaner